meta-ostro/classes/ostro-image.bbclass

```diff
@@ -280,6 +280,10 @@ NOHDD = "1"
 # the rootfs. WARNING: any change to this value will trigger a
 # rebuild (and re-sign, if enabled) of the combo EFI application.
 ROOTFS_PARTUUID_VALUE = "12345678-9abc-def0-0fed-cba987654321"
+# The second value is needed for the system installed onto
+# the device's internal storage in order to mount correct rootfs
+# when an installation media is still inserted into the device.
+ROOTFS_PARTUUID_VALUE2 = "12345678-9abc-def0-0fed-cba987654320"
```

Comment:
In hindsight I see that increasing the efi partitions' size is not really needed since they are populated from the data which can be found in /boot of the rootfs.
Will check if it can be easily avoided.